serving/src/main/java/ai/djl/serving/pyclient/PythonConnector.java

```diff
@@ -81,7 +83,18 @@ public Channel getChannel() {
         return this.channel;
     }
 
-    private static SocketConnector newInstance() {
-        return new SocketConnector();
+    private static PythonConnector newInstance() {
+        return new PythonConnector();
+    }
+
+    private boolean isUDS() {
+        ConfigManager configManager = ConfigManager.getInstance();
+        return configManager.useNativeIo();
+    }
+
+    private SocketAddress getSocketAddress() {
+        return isUDS()
+                ? new DomainSocketAddress(DEFAULT_SOCKET_PATH)
+                : InetSocketAddress.createUnresolved("127.0.0.1", 9000);
```

Comment:
Is it okay to have these private methods? I think these methods can be removed, if we include to ConnectorType as Python.